README.md

```diff
@@ -102,8 +102,8 @@ signature string: it either returns True or raises BadSignatureError.
 from ecdsa import SigningKey
 sk = SigningKey.generate() # uses NIST192p
 vk = sk.get_verifying_key()
-signature = sk.sign("message")
-assert vk.verify(signature, "message")
+signature = sk.sign("message".encode())
+assert vk.verify(signature, "message".encode())
```

Comment:
I think that changing it to byte string: `b"message"` would be less ambiguous what the intent is